Jetchat/app/src/main/java/com/example/compose/jetchat/profile/Profile.kt

```diff
@@ -84,31 +85,31 @@ fun ProfileScreen(userData: ProfileScreenState, onNavIconPressed: () -> Unit = {
                         modifier = Modifier
                             .clickable(onClick = {}) // TODO: Show not implemented dialog.
                             .padding(horizontal = 12.dp, vertical = 16.dp)
-                            .preferredHeight(24.dp)
+                            .preferredHeight(24.dp),
+                        contentDescription = null
                     )
                 }
             }
         )
-        WithConstraints {
-            Box(modifier = Modifier.weight(1f)) {
-                Surface {
-                    ScrollableColumn(
-                        modifier = Modifier.fillMaxSize(),
-                        scrollState = scrollState
-                    ) {
-                        ProfileHeader(
-                            scrollState,
-                            userData
-                        )
-                        UserInfoFields(userData, maxHeight)
-                    }
+        BoxWithConstraints(modifier = Modifier.weight(1f)) {
```

Comment:
Do you really need the `BoxWithConstraints` here? Not sure how the `maxHeight` is used below.

Reply:
I looked into it and I need it to comply with specs so that you scan scroll up in the profile without weirdness between landscape and portrait. Also I used it to fix landscape mode. `fillMaxHeight(0.5f)` did not work as I thought it would.

Reply:
Fair enough 👍